src/utils/hooks/useDateTimeFormat.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { useSelector } from "react-redux";
+import { languageSelector } from "../../reducers/settings.js";
+
+const useDateTimeFormat = (options?: Intl$DateTimeFormatOptions = {}) => {
+  const currentLanguage = useSelector(languageSelector);
+  const formatter = new Intl.DateTimeFormat(currentLanguage, options);
+
+  return formatter.format;
+};
```

Comment:
we should always use a cache. date formatting have an history of being a bottleneck in our app and this is why was had a component that is memo'ed in the past.

Reply:
moreover, it might be a blocker to do Intl if we want to go Hermes , https://github.com/facebook/hermes/issues/23

Reply:
Thanks for your comments!
In fact, after reading your responses from Slack I double-checked the Android supports and discovered that we have an issue with the configuration, `Intl` isn't injected as expected by our configuration. I have the feeling that the configuration which is currently located in `build.grade` files is outdated for this specific use case and should be updated as described [here](https://github.com/react-native-community/jsc-android-buildscripts#international-variant).

I recommend removing the commit related to `Intl` from this commit and open a new dedicated PR as the scope is bigger than expected.

@gre Regarding Android configuration from `build.gradle` files, is it the job of the core team to update it or can I put my hands in? I can see the configuration to run Intl from RNv0.59.x already in place, I think that we just have to bump it to supports version 0.60.x+.

Reply:
Here is the new [PR](https://github.com/LedgerHQ/ledger-live-mobile/pull/1858)